Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/Beckn/Confirm.hs

```diff
@@ -75,12 +83,7 @@ confirm transporterId (SignatureAuthResult _ subscriber) req =
                   void $
                     BP.sendRideAssignedUpdateToBAP dConfirmRes.booking ride
               DS.driverScoreEventHandler DST.OnNewRideAssigned {merchantId = transporterId, driverId = Id driverId}
-            DBooking.SpecialZoneBooking -> do
-              fork "on_confirm/on_update" $ do
-                handle (errHandler' dConfirmRes transporter) $ do
-                  onConfirmMessage <- ACL.buildOnConfirmMessage dConfirmRes
-                  void $
-                    BP.callOnConfirm dConfirmRes.transporter context onConfirmMessage
+            DBooking.RentalBooking -> undefined
```

Comment:
Lets complete these now

Reply:
Working on this @0utkarsh

Reply:
It will be the same as for `SpecialBooking` case (send `on_confirm`, but do not send `on_update`)